### PR TITLE
[CLOUD-3563] Update XP build configs to final repos / branches

### PR DESCRIPTION
**Pull request by luck3y**
https://issues.redhat.com/browse/CLOUD-3563
Signed-off-by: Ken WIlls <kwills@redhat.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request title is properly formatted: `[CLOUD-XYA] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

File: eap-xp/image.yaml [hunk 1]
```diff
@@ -39,8 +39,8 @@ modules:
                   ref: EAP_731_DEV1
           - name: jboss-eap-7-image
             git:
-                  url: https://github.com/jboss-container-images/jboss-eap-7-image.git
-                  ref: EAP_731_DEV1
+                  url: https://github.com/luck3y/jboss-eap-7-image.git
```
Comment (luck3y) [inline eap-xp/image.yaml]: This will need to be updated when we have the final CR artifacts.

File: eap-xp/image.yaml [hunk 2]
```diff
@@ -120,7 +120,7 @@ modules:
 artifacts:
   - name: maven-repo
     target: maven-repo.zip
-    md5: 0b8c40d5d53e06b4fc02977f55146216
+    md5: f0586825b5828e9f1a4050d3b665a63f
```
Comment (luck3y) [inline eap-xp/image.yaml]: This will need to be updated when we have the final CR artifacts.

**Comment by luck3y** [comment]
@dbkreling @rgdoliveira FYI this is a placeholder PR until we have final artifacts.

It also needs: https://github.com/jboss-container-images/jboss-eap-7-openshift-image/pull/357 merged first.

**Review by dbkreling — APPROVED** [review]
LGTM

**Pull request merged**